tox.ini

```diff
@@ -34,6 +34,9 @@ commands =
     coverage run -p -m pytest --durations=20 {posargs:tests}
     coverage combine
     coverage xml
+deps =
```

Comment:
by default this will install the CUDA 10.2 version (and then use the CPU implementations since the GHA worker does not have CUDA devices).

We could use the CPU-only build instead, which is considerably smaller. This can be done by setting an `--extra-index-url`, cf. cf. https://pytorch.org/get-started/locally/

```console
pip3 install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cpu
```

Reply:
cf. https://github.com/mberr/torch-ppr/pull/14/commits/6cf39bbb019ef92bf11f116017b1ff85c4113d74